plugins/postcss-is-pseudo/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@csstools/postcss-is-pseudo",
+  "version": "0.1.0",
```

Comment:
Is it intended to be experimental?

Reply:
I mostly do a `0.1.0` release first for other projects?
Not sure if there is an existing convention here?

It should not be placed in `/experimental/` because there is no existing plugin.